### ISSUE TITLE
Erratic loss pattern when training both wan 2.2 models at the same time

### ISSUE THREAD

**Issue by StormDog2** [kohya-ss/musubi-tuner#926]
Hey, I've been trying to train multiple loras in dual mode and found a strange issue. The loss value was jumping up and down erratically through out the whole training. The results, while not catastrophic, were subpar at best with the concept not properly working 50% of the time. I've used the same dataset and captions for training low and high separately with great success so I'm not sure what the issue is. 

**Environment details:**
Musubi-tuner: Latest commit (Commit 0b2d692)
Operating System: Linux
GPU model and VRAM: RTX Pro 6000 96 Gb Vram
Python version: 3.11 
PyTorch version: 2.8
CUDA version 12.8.1

**Error message:** None

**Expected behaviour:** The model learns the concept of the dataset and loss goes down
**Actual behaviour:** The model partially learns the concept. Loss goes up and down randomly in big spikes.

**Training config** (2 runs; same behaviour):

First training:

accelerate launch --num_cpu_threads_per_process 1 src/musubi_tuner/wan_train_network.py --task t2v-A14B --dit /workspace/musubi-tuner/models/diffusion_models/split_files/diffusion_models/wan2.2_t2v_low_noise_14B_fp16.safetensors --dit_high_noise /workspace/musubi-tuner/models/diffusion_models/split_files/diffusion_models/wan2.2_t2v_high_noise_14B_fp16.safetensors --vae /workspace/musubi-tuner/models/vae/split_files/vae/wan_2.1_vae.safetensors --t5 /workspace/musubi-tuner/models/text_encoders/models_t5_umt5-xxl-enc-bf16.pth --dataset_config /workspace/musubi-tuner/dataset/dataset.toml --mixed_precision fp16 --fp8_base --fp8_scaled --sdpa --optimizer_type adamw --learning_rate 2e-5 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --network_alpha 1 --timestep_sampling sigmoid --discrete_flow_shift 4.0 --lr_scheduler cosine --lr_warmup_steps 100 --network_args "loraplus_lr_ratio=4" --network_dropout 0.1  --save_state --save_every_n_epochs 4 --max_train_epochs 200 --preserve_distribution_shape --min_timestep 0 --max_timestep 1000 --timestep_boundary 875 --logging_dir=logs --log_prefix=Test --output_name Test --output_dir /workspace/musubi-tuner/output 

Second training:

accelerate launch --num_cpu_threads_per_process 1 src/musubi_tuner/wan_train_network.py --task t2v-A14B --dit /workspace/musubi-tuner/models/diffusion_models/split_files/diffusion_models/wan2.2_t2v_low_noise_14B_fp16.safetensors --dit_high_noise /workspace/musubi-tuner/models/diffusion_models/split_files/diffusion_models/wan2.2_t2v_high_noise_14B_fp16.safetensors --vae /workspace/musubi-tuner/models/vae/split_files/vae/wan_2.1_vae.safetensors --t5 /workspace/musubi-tuner/models/text_encoders/models_t5_umt5-xxl-enc-bf16.pth --dataset_config /workspace/musubi-tuner/dataset/dataset.toml --mixed_precision fp16 --fp8_base --fp8_scaled --sdpa --optimizer_type adamw --learning_rate 3e-5 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --network_alpha 16 --timestep_sampling qinglong_flux --discrete_flow_shift 4.0 --lr_scheduler cosine --lr_warmup_steps 100 --network_args "loraplus_lr_ratio=4" --network_dropout 0.1  --save_state --save_every_n_epochs 4 --max_train_epochs 200 --preserve_distribution_shape --min_timestep 0 --max_timestep 1000 --timestep_boundary 875 --logging_dir=logs --log_prefix=Test --output_name Test --output_dir /workspace/musubi-tuner/output 

Dataset (same for both):

resolution, caption_extension, batch_size, num_repeats, enable_bucket, bucket_no_upscale should be set in either general or datasets
otherwise, the default values will be used for each item

general configurations
[general]
caption_extension = ".txt"
batch_size = 1
enable_bucket = true
bucket_no_upscale = false

[[datasets]]
video_directory = "/workspace/musubi-tuner/dataset"
cache_directory = "/workspace/musubi-tuner/dataset/cache1"
resolution = [896, 504]  # 921,600 pixels
frame_extraction = "full"
max_frames = 33

other datasets can be added here. each dataset can have different configurations

Thanks for your time.
